query/src/common/meta/meta_client.rs

```diff
@@ -46,7 +46,7 @@ impl MetaClientProvider {
 
     /// Get kv async client, operations trait defined in KVApi.
     pub async fn try_get_kv_client(&self) -> Result<Arc<dyn KVApi>> {
-        let local = self.grpc_conf.meta_service_config.address.is_empty();
+        let local = self.grpc_conf.kv_service_config.address.is_empty();
```

Comment:
Considering https://github.com/datafuselabs/databend/issues/5323

Will we adopt `kv_service_config` always and remove `meta_service_config`?